resources/applicationautoscaling-scalable-targets.go

```diff
@@ -52,13 +52,22 @@ func (l *ApplicationAutoScalingScalableTargetLister) List(_ context.Context, o i
 			}
 
 			for _, out := range resp.ScalableTargets {
+				tags, err := svc.ListTagsForResource(
+					&applicationautoscaling.ListTagsForResourceInput{
+						ResourceARN: out.ScalableTargetARN,
+					})
+				if err != nil {
+					return nil, err
```

Comment:
Thanks. 

This needs to log an error only though instead of interrupting the discovery.  I had to make this call a while back to not interrupt the the discovery because there seem to be some orgs that have tight permissions.

This will need to be done for all tags going forward.

If there is no error it should set a predefined variable and then use that to set the tags on the object.

Reply:
Makes sense. I've updated my commit.